src/geocat/viz/util.py

```diff
@@ -1068,6 +1200,75 @@ def plotELabels(da: xr.DataArray,
                 fontsize: int = 22,
                 whitebbox: bool = False,
                 horizontal: bool = True) -> list:
+    """This function is deprecated. Please use `plot_extrema_contour_labels`
```

Comment:
Despite the confusing docs, these look to be extrema locations/points rather than contours.  Maybe we change it to `plot_extrema_labels`?  

I wonder if we should changes `clabel_locations` as well while we're at it.

Reply:
I like 'plot_extrema_labels`

What would you change `clabel_locations` to? just `label_locations`? I agree that if we're going to change the kwarg it should be now.

Reply:
`label_locations` sounds reasonable to me.  

`label_coords` or `label_coordinates` are other options that come to mind, but I don't know that they are that much more clear.  And with `clabel_locations` for the other function `label_locations` makes sense (guessing we don't want to change everything here).